external/java/src/main/java/ch/epfl/dedis/lib/omniledger/Proof.java

```diff
@@ -45,8 +55,12 @@ public Proof(OmniLedgerProto.Proof p) {
      * hashes
      * @throws CothorityException
      */
-    public boolean verify() throws CothorityException {
-        return false;
+    public boolean verify(SkipblockId id) throws CothorityException {
+        if (!isOmniledgerProof()){
+            return false;
+        }
+        // TODO: more verifications
```

Comment:
Is there any open issue related to this missing checks in verify method?

Reply:
https://github.com/dedis/cothority/issues/1440